src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/client/PpudAutomationApiClient.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.client
+
+import io.micrometer.core.instrument.Counter
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.MediaType
+import org.springframework.web.reactive.function.BodyInserters
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.PpudSearchRequest
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions.PpudSearchResponse
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.exception.ClientTimeoutException
+import java.time.Duration
+import java.util.concurrent.TimeoutException
+
+class PpudAutomationApiClient(
+  private val webClient: WebClient,
+  @Value("\${ndelius.client.timeout}") private val ppudAutomationTimeout: Long,
```

Comment:
Probably needs its own timeout rather than reusing the ndelius client

Reply:
Tks - Idea hid this from me.  Good catch.